### ISSUE TITLE
PSR2.Methods.FunctionCallSignature.Indent cannot be exclude from command line

### ISSUE THREAD

**Issue by bhennesAdv** [squizlabs/PHP_CodeSniffer#3920]
## Describe the bug
The following rule is reported as an error from phpcs but cannot be excluded by the --exclude parameter :
PSR2.Methods.FunctionCallSignature.Indent

### Code sample
```php
<?php
class TestClass
{
    public function testMethod(string $testParam): void
    {
        echo $testParam;
    }
};

$object = new TestClass();
$object->testMethod(
 'testParam1' //  <== Error is here
);
```

### Custom ruleset
```xml
<?xml version="1.0"?>
<ruleset name="My Custom Standard">
	<rule ref="PSR2.Methods.FunctionCallSignature.Indent">
		<severity>5</severity>
	</rule>
</ruleset>
```

### To reproduce
1. Create a file called `test.php` with the code sample above.
2. Create a file called `custom-ruleset.xml` with the ruleset sample above.
3. Run `phpcs test.php --standard=custom-ruleset.xml test.php -s`
4. See that the error is detected (So it's active and known) :
```bash
FILE: /magento/test.php
--------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------
 12 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 1
    |       |     (PSR2.Methods.FunctionCallSignature.Indent)
--------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------

Time: 10ms; Memory: 4MB
```
5. Run `phpcs --standard=custom-ruleset.xml test.php -s --exclude=PSR2.Methods.FunctionCallSignature.Indent`
6. See the error message below :
```bash
ERROR: The specified sniff code "PSR2.Methods.FunctionCallSignature.Indent" is invalid

Run "phpcs --help" for usage information
```

## Expected behavior
The rule should be ignored and the output clean

## Versions (please complete the following information)

| | |
|-|-|
| Operating System | Ubuntu 20.04
| PHP version | 7.4, 8.1 (both tested)
| PHP_CodeSniffer version | 3.3.1, 3.7.2 (both tested)
| Standard | PSR2
| Install type | composer, PHAR archive (both tested)

## Please confirm:

- [ X] I have searched the issue list and am not opening a duplicate issue.
- [ X] I confirm that this bug is a bug in PHP_CodeSniffer and not in one of the external standards.
- [ X] I have verified the issue still exists in the `master` branch of PHP_CodeSniffer. (Last release tested, not tested from code)

**Comment by bhennesAdv** [comment]
Hello,

Thank you, I had no idea about this difference.
Sorry about that.
